backend/gn_module_monitoring/command/utils.py

```diff
@@ -116,6 +116,137 @@ def process_available_permissions(module_code, session):
         )
 
 
+def clean_existing_permissions(module_code, session):
```

Comment:
It would be beneficial to add logging to the `clean_existing_permissions` function. This would help in tracking the progress of the function and identifying any issues that might occur. [medium]